htdocs/install/mysql/tables/llx_product_attribute_extrafields.key.sql

```diff
@@ -0,0 +1,19 @@
+-- ===================================================================
```

Comment:
I suggest to rename the table into
llx_product_attribute_extrafields so the table will be create only if we activate the module variant.

Reply:
rename the table or the file ?
llx_product_attribute_extrafields.sql to llx_product_attribute_extrafields-variants.sql

Reply:
> rename the table or the file ? llx_product_attribute_extrafields.sql to llx_product_attribute_extrafields-variants.sql

You're right, i mean rename the "file" only.

Reply:
Does I need to remove migrations ?